main.cpp

```diff
@@ -64,7 +64,12 @@ int main(int argc, char *argv[]) {
 		path = "./";
```

Comment:
Could you reformat the code to help the reading of the functions ? They sometimes are on 2 lines.
I think it should be better to put the arguments on a second line.